src/Auth/Eloquent/UserGroupModel.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Statamic\Auth\Eloquent;
+
+use Illuminate\Database\Eloquent\Model as Eloquent;
+
+class UserGroupModel extends Eloquent
+{
+    protected $guarded = [];
+
+    protected $table = 'groups';
+
+    protected $casts = [
+        'roles' => 'json',
+        'created_at' => 'datetime',
+        'updated_at' => 'datetime',
+    ];
+
+    public function __construct(array $attributes = [])
+    {
+        parent::__construct($attributes);
+
+        if ($table = config('statamic.users.groups')) {
```

Comment:
```suggestion
        if ($table = config('statamic.users.tables.groups')) {
```

Reply:
@ryanmitchell I had to make this change to be able to log in but after this things seem to work smoothly.